subtitles.go

```diff
@@ -246,6 +246,7 @@ type StyleAttributes struct {
 	WebVTTVertical       string
 	WebVTTViewportAnchor string
 	WebVTTWidth          string
+	WebVTTCSS            string
```

Comment:
I think that would be best to make this attribute a `[]string` to store each line separately

Reply:
:+1: